README.rst

```diff
@@ -36,16 +36,14 @@ The required dependencies to use the software are:
 
 * Python >= 2.6,
 * setuptools
-* Numpy >= 1.6
+* Numpy >= 1.6.1
 * SciPy >= 0.9
 * Scikit-learn >= 0.12.1
-* Nibabel >= 1.1.0.
+* Nibabel >= 1.3.0
```

Comment:
Note: we don't support 12.04 anymore (because of numpy / scipy). We only support 12.04 neurodebian. This way, we can upgrade most of packages.
From 1.1.0 to 1.3.0, a lot of significant bugs were fixed. Also, some code to handle affine transformations was added. We can rely on it and thus simplify nilearn's code.